device.go

```diff
@@ -122,13 +123,11 @@ func getPCIDeviceName(s *sandbox, pciID string) (string, error) {
 		}
 	}
 
-	// Check if the PCI path is present before we setup the pci device map.
-	_, err = os.Stat(filepath.Join(rootBusPath, pciAddr))
-	if err == nil {
-		// Found pci path. It's there before we wait for the device map.
-		// We cannot find the name for it.
-		fieldLogger.Info("Device found on pci device path")
-		return "", nil
+	// Rescan pci bus if we need to wait for a new pci device
+	if err = ioutil.WriteFile(pciBusRescanPath, []byte{'1'}, 0200); err != nil {
```

Comment:
please use `pciBusMode`

Reply:
or could you implement a new function here to rescan the PCI bus and use it in `grpc.go` ?